### PR TITLE
feat(oui-back-button): add oui-back-button component.

### PR DESCRIPTION
**Pull request by frenautvh**
http://angular_feat_back_button.ui-kit.ovh/#!/oui-angular/back-button

### CODE REVIEW

File: packages/oui-back-button/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Back button
+
+<component-status cx-design="complete" ux="prototype"></component-status>
+
+## Usage
+
+```html
+<oui-back-button>
```
Comment (jimmyfortinx) [inline packages/oui-back-button/README.md]: Need to be updated to list all attributes.

File: packages/oui-back-button/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+</oui-back-button>
+```
+
+## Examples
+
+### Default
+
+```html:preview
+<oui-back-button>
```
Comment (jimmyfortinx) [inline packages/oui-back-button/README.md]: The back button (on CX specs) also includes the text at the right, do we want it ?

File: packages/oui-back-button/src/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  let $compile
+  let $window
+
+  beforeEach(angular.mock.module('oui.back-button'))
+
+  beforeEach(inject((_$rootScope_, _$compile_, _$window_) => {
+    $rootScope = _$rootScope_
+    $compile = _$compile_
+    $window = _$window_
+    $window.history = {
```
Comment (jimmyfortinx) [inline packages/oui-back-button/src/index.spec.js]: You should use `spyOn($window.history, 'back')` instead

File: packages/oui-back-button/README.md (hunk 1 above)
Comment (neolitec) [reply packages/oui-back-button/README.md]: Too general name for button that must be placed at the left of an h2 top positioned title I think.

Comment (frenautvh) [reply packages/oui-back-button/README.md]: if we had the h2 title yes it makes sense to rename it

File: packages/oui-back-button/README.md (hunk 2 above)
Comment (frenautvh) [reply packages/oui-back-button/README.md]: not sure about this one, do we really want to restrict the component to be used with a h2? (similar to : do we add labels to input components)
if that's the case, it won't be a oui-back-button anymore but maybe something like oui-page-title (please give me great ideas ;) )

Comment (jimmyfortinx) [reply packages/oui-back-button/README.md]: Saw together, we will keep it simple.

**Pull request merged**